app/bundles/LeadBundle/Form/Type/EntityFieldsBuildFormTrait.php

```diff
@@ -169,106 +187,69 @@ function (FormEvent $event) use ($alias, $type) {
 
                     $builder->add($alias, $type, $opts);
                     break;
-                case 'select':
-                case 'multiselect':
-                case 'boolean':
-                    if ('multiselect' == $type) {
```

Comment:
Seems we lost these in the code change - is that because these field types are no longer maxed out at 255 chars?

Reply:
Yes, I added the constraint to the MultiselectType directly:

https://github.com/mautic/mautic/pull/8237/files#diff-efc607e77428fede0c8fb86c4e93d4a9R32